### DISCUSSION TITLE
Skip vs Next Review

### DISCUSSION THREAD

**Discussion by tamihuber** [paulwinterbauer/NBIC#355] [Q&A]
is there a difference? Does one lock the form and the other 'excepts' you from that vessel?

**Comment by paulwinterbauer** [comment]
Next Review will only take you to the next if you've completed the current review, otherwise you'll end up on the same report/amend check.

Skip allows you to leave the current review and move to the next without completing it.  It will stay in the queue for another data manager to pick up.

**Comment by tamihuber** [comment]
I mean skip in the vessel history. Not the report.

**Reply by paulwinterbauer** [reply]
Same idea.  If there's a pending duplicate check in the vessel history, clicking next review will just return you to the same vessel whereas skip would allow you to move on and leave the pending for another data manager.

**Comment by tamihuber** [comment]
ah. Gotcha. Thanks!

**Comment by tamihuber** [comment]
Next review on a normal basis (from vessel history page) takes you to the next pending review, not the next report in that same vessel. Interesting. So what happens if there's no other form review_pending in that vessel and you hit "next review"?

**Reply by paulwinterbauer** [reply]
It depends which queue you picked on the dashboard page, but generally the next pending report overall ordered by time received.

**Comment by tamihuber** [comment]
So right, we were in Pending Reviews (not the All tab, just the Review tab), and had to go into vessel history. Since we're training, but not yet doing amendment checks, we're skipping those. Chris mentioned today that "Next review" does the same thing as skip. He showed me and it did seem that way, so I told him I'd ask you the difference.

**Comment by tamihuber** [comment]
I'm still not clear how Skip would differ from Next Review in this case....if you say the next pending report would come in. So the vessel itself has nothing to do with it, it seems, and hitting either results in teh same thing.Not a biggie, but I couldn't answer his question if there is a difference ;).

**Reply by paulwinterbauer** [reply]
I think I see.  If you're in the review queue from the dashboard, you're cycling through the reports that are rev_pend by time received.  I kinda shoehorned the amendment check onto the end of the review process, but when it goes to the amendment checker for that report the report has been moved to dup_pend at that point.  So in this scenario next review and skip would do the same thing, because next review would give you the next rev_pend report.  

In general though, skip is for when you get a task that you don't want to complete and want to leave for someone else.

**Comment by tamihuber** [comment]
Perfect! And ya, we wanted to skip the amendment checking, because of the training situation (where we're not doing those yet). Good to know!
